cpp/src/renumber.cuh

```diff
@@ -193,7 +199,8 @@ namespace cugraph {
 			      T_in ** numbering_map,
 			      int max_threads_per_block = CUDA_MAX_KERNEL_THREADS,
 			      int max_blocks = CUDA_MAX_BLOCKS,
-			      uint32_t hash_size = 8191) {
+			      detail::hash_type hash_size = 8191) {
```

Comment:
Is this default size working for the 100k vertices test on GV100 only or do you expect it to be pretty good for all sizes/cards?

Reply:
I don't know.  I expect we'll want to do some more extensive experimentation.  In my testing thus far it seems to be a good value.  But I expect that using strings will change the dynamic a bit, and I'm hoping that I can optimize the segmented sort a bit more and use thet here - which may also change the optimal hash size.

Reply:
We should probably create an issue to keep this on the radar. It would be great to have some heuristics to set it automatically based on the input. Renumbering is not cheap and this can impact perf by over 2x.

Reply:
#243